oidc-primer.html

```diff
@@ -0,0 +1,1219 @@
+   <p>Using the <code>kid</code> value in the ID token, the AS searches the OP public keys for one that matches the
+one used to sign the ID token. If no public key is found, the AS must reject the request.</p>
+   <h4 class="no-num heading settled" id="request-flow-step-13"><span class="content">13. Access Token Response</span><a class="self-link" href="#request-flow-step-13"></a></h4>
+   <p>Now that the AS has performed all of its checks, we can trust that the agent in the <code>webid</code> claim (<code>https://alice.coolpod.example/profile/card#me</code>) is the same agent on whose behalf the
+request was made. Using that information, the AS performs authorization and returns access token.</p>
+   <p>Response:</p>
+<pre class="language-json highlight"><c- p>{</c->
+   <c- f>"access_token"</c-><c- p>:</c-><c- u>"sbjsbhs(/SSJHBSUSSJHVhjsgvhsgvshgsv"</c-><c- p>,</c->
+   <c- f>"token_type"</c-><c- p>:</c-><c- u>"Bearer"</c->
+<c- p>}</c->
+</pre>
+   <h4 class="no-num heading settled" id="request-flow-step-14"><span class="content">14. Sends request with Access Token</span><a class="self-link" href="#request-flow-step-14"></a></h4>
+   <p class="note" role="note"><span>Note:</span> In practice client would need to upgrade RPT by pushing
+authorization related claim. This primer only focuses on authentication so we omit
+this detail here.</p>
+   <p>With the access token, we’re ready to make our request.</p>
+<pre class="language-http highlight">GET https://bob.otherpod.example/private/photo_album.ttl
+Headers: {
+    authorization: "Bearer sbjsbhs(/SSJHBSUSSJHVhjsgvhsgvshgsv",
+}
+</pre>
+   <h4 class="no-num heading settled" id="request-flow-step-15"><span class="content">15. Returns Result</span><a class="self-link" href="#request-flow-step-15"></a></h4>
+   <p>Given all went well, the RS should return the requested content.</p>
+  </main>
+  <div data-fill-with="conformance">
+   <h2 class="no-ref no-num heading settled" id="w3c-conformance"><span class="content">Conformance</span><a class="self-link" href="#w3c-conformance"></a></h2>
+   <h3 class="no-ref no-num heading settled" id="w3c-conventions"><span class="content">Document conventions</span><a class="self-link" href="#w3c-conventions"></a></h3>
+   <p>Conformance requirements are expressed
+    with a combination of descriptive assertions
+    and RFC 2119 terminology.
+    The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL”
+    in the normative parts of this document
+    are to be interpreted as described in RFC 2119.
+    However, for readability,
+    these words do not appear in all uppercase letters in this specification. </p>
+   <p>All of the text of this specification is normative
+    except sections explicitly marked as non-normative, examples, and notes. <a data-link-type="biblio" href="#biblio-rfc2119">[RFC2119]</a> </p>
```

Comment:
I'm a bit confused here, as primers tend to be non-normative supplements to the normative spec, so if both these documents are normative, then the relationship between them needs to be described carefully.

Reply:
The primer is _all_ non-normative. This is boilerplate that was likely auto-generated. I'll look into adjusting this.

Reply:
Good!